### PR TITLE
Added functions for trips

### PR DESCRIPTION
**Pull request by pavan3008**
# Proposed Changes

Added function for trips that helps to perform CRUD operations.

# Checklist

- [x]  This has been tested
- [x]  Self-documenting code
- [x]  Code is linted

# Instructions to review

Please review the code

### CODE REVIEW

File: src/Trips.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
+const AWS = require('aws-sdk');
+
+const dynamoDB = new AWS.DynamoDB.DocumentClient();
+const { v4: uuidv4 } = require('uuid');
+
+const DYNAMODB_TABLE_NAME = process.env.DYNAMODB_TABLE_NAME;
+
+async function getTripsForUser(userId) {
+  // Get all trips details for a user
```
Comment (NITHISHrajapuram) [inline src/Trips.js]: To be consistent with other functions you can remove this comment.

File: src/Trips.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+        },
+        UpdateExpression: updateExpression,
+        ExpressionAttributeValues: expressionAttributeValues,
+        ReturnValues: 'ALL_NEW',
+      };
+      const updatedItem = await dynamoDB.update(params).promise();
+      return updatedItem.Attributes; // return only the updated attributes
+    });
+    const updatedItems = await Promise.all(updatePromises);
+    return { statusCode: 200, body: JSON.stringify(updatedItems) };
```
Comment (NITHISHrajapuram) [inline src/Trips.js:155]: spacing

File: src/Trips.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+
+  try {
+    await dynamoDB.batchWrite(params).promise();
+    const response = {
+      tripId: tripId.replace('Trip', ''),
+      userId,
+      trip_name,
+      trip_status,
+    };
+    return { statusCode: 201, body: JSON.stringify(response) };
```
Comment (NITHISHrajapuram) [inline src/Trips.js:103]: required a space to maintain consistency

**Review by NITHISHrajapuram — CHANGES_REQUESTED** [review]

**Review by NITHISHrajapuram — APPROVED** [review]
Good to go.

**Comment by pavan3008** [comment]
Thanks!

**Pull request merged**